### DISCUSSION TITLE
How can I include static files as part of the binary

### DISCUSSION THREAD

**Discussion by goldenratio** [actix/examples#658] [Q&A] [answered]
Hi all,

How can I include static files as part of the binary? is it possible to add this use case to the examples?

**Comment by robjtede** [comment, marked as answer]
Hey! Yes, the crate you're looking for is [`actix-web-static-files`](https://docs.rs/actix-web-static-files).

It'd be great to get an example put up in here, too. If you're up for it just submit a PR 👍🏻
